README.md

```diff
@@ -66,7 +66,7 @@ $ cd cluster-lifecycle-e2e
 $ cp pkg/resources/options_template.yaml resources/options.yaml
 ```
 
-3. If you want run the "import" scenario, copy the kubeconfig of the cluster to import to resources/import/kubeconfig and set the kubeconfig of the cluster in the options.yaml to `/opt/.kube/import-kubeconfig`
+3. If you want run the "import" scenario, copy the kubeconfig of the cluster to import to pkg/tests/resources/hub/import/kubeconfig and set the kubeconfig of the cluster in the options.yaml to `/opt/.kube/import-kubeconfig`
```

Comment:
The path `resources/import` does not exist in the repo, so I replace it with `pkg/tests/resources/hub/import`, not sure if it is appropriate.